src/components/Phase/Phase.tsx

```diff
@@ -181,11 +199,7 @@ const Phase = ({
                   <Text variant="p" size="sm" fontBold="medium">
                     {item.label}
                   </Text>
-                  {item.completed ? (
-                    <CheckCircleIcon className="w-4 h-4 text-green-500" />
-                  ) : (
-                    <ExclamationCircleIcon className="w-4 h-4 text-yellow-600" />
-                  )}
+                  {iconStatus(item.status)}
```

Comment:
y aquí ya podrías hacer algo como

```suggestion
                  {ICON_STATUS[item.status]}
```